src/poll/responser.rs

```diff
@@ -0,0 +1,666 @@
+// TODO: this `#![allow(unused)]` lint is temporarily added. Will be removed sooner.
+#![allow(unused)]
+use std::collections::HashMap;
+use std::sync::RwLock;
+use std::{net::Ipv4Addr, slice::from_raw_parts_mut, sync::Arc, thread::spawn};
+
+use bitfield::bitfield;
+use libc::IFA_ADDRESS;
+use lockfree::queue::Queue;
+
+use crate::qp::QpContext;
+use crate::{device::QpType::RawPacket, Sge};
+use crate::{
+    device::{
+        ToCardWorkRbDescBuilder, ToCardWorkRbDescCommon, ToHostWorkRbDescAethCode,
+        ToHostWorkRbDescOpcode, ToHostWorkRbDescRead,
+    },
+    Device, Error,
+};
+
+/// A single MR that contains a free list of packet buffer. Each slot is 64 Byte.
+pub(crate) const ACKNOWLEDGE_BUFFER_SLOT: usize = 64;
```

Comment:
use _CNT as suffix to explicitly show it's a number, and the comment in the above line is 64, but the bellow line is 1024.

if the const's name is clear, no comment is needed.

Reply:
Got it.

Reply:
And I think it may be better to move the constant inside the `AcknowledgeBuffer`.